### ISSUE TITLE
Improve error message when augmenting with a sha256 tag

### ISSUE THREAD

**Issue by ewels** [seqeralabs/wave#477]
As documented in https://github.com/seqeralabs/wave/pull/448, Wave cannot support SHA256 tags when augmenting an image, because the image itself is changing.

When someone attempts to do this, the following error is returned:

```
Container requests made using a SHA256 as tag does not support the 'containerConfig' attribute
```
(I'm not sure where this error message is coming from, as I can't find it in the source code of the Wave or Nextflow repos).

This error mode is likely going to be quite common as people take existing pipelines with fixed SHA256 tags and enable Wave. It would be great if we could improve the error message to (a) make it easier to understand and (b) explain how to fix the issue - point to the docs.

Basically, no-one getting this error is going to be trying to use the 'containerConfig' attribute intentionally (I don't know even what that is). So we should instead throw an error message about SHA256 tags and Wave augmentation.

**Comment by pditommaso** [comment]
This is solved, right?

**Comment by munishchouhan** [comment]
@ewels error message comes from here
https://github.com/seqeralabs/wave/blob/32787158d54dfe41675fedec083c07cd18ac5dbd/src/main/groovy/io/seqera/wave/controller/ContainerController.groovy#L379

this is containerconfig 
https://github.com/seqeralabs/wave/blob/32787158d54dfe41675fedec083c07cd18ac5dbd/src/main/groovy/io/seqera/wave/controller/ContainerController.groovy

I suppose the error meant that container augmentation is allowed when SHA256 tag is provided

@pditommaso should we change the error message?
